src/new-client/src/Web/TearOutSection/handleTearOutSection.web.ts

```diff
@@ -0,0 +1,29 @@
+import { constructUrl } from "@/utils/url"
+import { openWindow } from "@/utils/window/openWindow"
+import { tearOutSection } from "./state"
+
+export function handleTearOutSection(section: string) {
```

Comment:
We should create a type for section throughout this functionality so we remove the ability to use a string with a typo.

Reply:
Done, for now it's `type Section = "liverates" | "trades" | "analytics"`